### PR TITLE
rename sandbox to extensions  - manual

### PR DESCRIPTION
**Pull request by davidhadas**
manual renaming

### CODE REVIEW

File: config/redirects.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ plugins:
       community/contributing/contributing.md: https://github.com/knative/community/blob/main/CONTRIBUTING.md
       community/contributing/governance.md: https://github.com/knative/community/blob/main/GOVERNANCE.md
       community/contributing/mechanics.md: https://github.com/knative/community/tree/main/mechanics
-      community/contributing/mechanics/creating-a-sandbox-repo.md: https://github.com/knative/community/blob/main/mechanics/CREATING-A-SANDBOX-REPO.md
+      community/contributing/mechanics/creating-a-extensions-repo.md: https://github.com/knative/community/blob/main/mechanics/CREATING-A-SANDBOX-REPO.md
```
Comment (psschwei) [inline config/redirects.yml:59]:
```suggestion
      community/contributing/mechanics/creating-an-extensions-repo.md: https://github.com/knative/community/blob/main/mechanics/CREATING-A-SANDBOX-REPO.md
```

should we also rename the file to `CREATING-AN-EXTENSIONS-REPO`?

Comment (davidhadas) [reply config/redirects.yml]: Fixed also A to AN and the actual link

File: docs/community/governance.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ election process for our technical oversight committee.
 - [Repository Guidelines](https://github.com/knative/community/blob/main/REPOSITORY-GUIDELINES.md):
 how we create and remove core repositories.
 
-- [Sandbox repo process](https://github.com/knative/community/blob/main/mechanics/CREATING-A-SANDBOX-REPO.md):
+- [Extensions repo process](https://github.com/knative/community/blob/main/mechanics/CREATING-A-SANDBOX-REPO.md):
```
Comment (psschwei) [inline docs/community/governance.md:68]:
```suggestion
- [Extensions repo process](https://github.com/knative/community/blob/main/mechanics/CREATING-AN-EXTENSIONS-REPO.md):
```

File: docs/eventing/custom-event-source/custom-event-source/README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ This type of integration requires more effort than using some simpler integratio
 !!! note
     If you have created a new event source type that is not a part of the core Knative project, you can open a pull request to add it to the list of [Third-Party Sources](../../sources/#third-party-sources), and announce the new source in the [#knative-eventing Slack channel](https://cloud-native.slack.com/archives/C04LMU33V1S).
 
-    You can also add your event source to the [`knative-extensions`](https://github.com/knative-extensions) organization, by following the instructions to [create a sandbox repository](https://github.com/knative/community/blob/main/mechanics/CREATING-A-SANDBOX-REPO.md).
+    You can also add your event source to the [`knative-extensions`](https://github.com/knative-extensions) organization, by following the instructions to [create a extensions repository](https://github.com/knative/community/blob/main/mechanics/CREATING-A-SANDBOX-REPO.md).
```
Comment (psschwei) [inline docs/eventing/custom-event-source/custom-event-source/README.md:10]:
```suggestion
    You can also add your event source to the [`knative-extensions`](https://github.com/knative-extensions) organization, by following the instructions to [create a extensions repository](https://github.com/knative/community/blob/main/mechanics/CREATING-AN-EXTENSIONS-REPO.md).
```

**Review by psschwei — COMMENTED** [review]
Probably should wait to merge this until after https://github.com/knative/community/pull/1419 gets in (for the page rename)

**Comment by davidhadas** [comment]
> Probably should wait to merge this until after [knative/community#1419](https://github.com/knative/community/pull/1419) gets in (for the page rename)

/hold

**Review by psschwei — COMMENTED** [review]
/lgtm
/approve

**Comment by davidhadas** [comment]
/unhold
https://github.com/knative/community/pull/1419 merged

**Pull request merged**